CHANGELOG_PENDING.md

```diff
@@ -1,5 +1,8 @@
 ### Improvements
 
+- Allow lowercase `type` on config entries.
+  [#201](https://github.com/pulumi/pulumi-yaml/pull/201)
```

Comment:
we use the pull request number rather than the issue number. I'm not sure why we do, but we do.